test/features/support/hooks.js

```diff
@@ -0,0 +1,96 @@
+// Before And After hooks used while feature executes
```

Comment:
code style needs to align with standardJS also use const/let instead of var accordingly